app/js/components/patient.jsx

```diff
@@ -10,6 +10,7 @@ import React from 'react';
 import Collapsible from 'react-collapsible';
 import apiCall from '../utilities/apiHelper';
 import Addresses from './addresses'
+import Identifiers from './Identifiers';
```

Comment:
In the above './Identifiers', did you change the convention to start with capital I?